grafana/greptimedb.json

```diff
@@ -2387,13 +2501,13 @@
     "list": []
   },
   "time": {
-    "from": "now-3h",
+    "from": "now-30m",
     "to": "now"
   },
   "timepicker": {},
   "timezone": "",
   "title": "GreptimeDB",
   "uid": "e7097237-669b-4f8d-b751-13067afbfb68",
-  "version": 9,
+  "version": 12,
   "weekStart": ""
 }
```

Comment:
Add an empty line?

Reply:
I've checked the raw exported JSON from grafana doesn't contain a tailing empty line. So I keep it as is